app/Console/Commands/AddExpression.php

```diff
@@ -48,7 +48,8 @@ public function handle()
         $expression_key = explode('.', $this->argument('key'))[1];
         $expression_value = $this->argument('value');
         $reviewed = $this->option('force');
-        $expressions = require base_path('resources/lang/'.$language.'/'.$file.'.php');
+        $path = 'resources/lang/'.$language.'/'.$file.'.php';
+        $expressions = file_exists($path) ? require base_path($path) : [];
```

Comment:
Fixed bug on locale:add artisan command causing error if the target file does not exist.